pkg/apis/db/v1alpha1/cassandracluster_types.go

```diff
@@ -682,9 +683,10 @@ type CassandraClusterSpec struct {
 	MaxPodUnavailable int32 `json:"maxPodUnavailable"` //Number of MaxPodUnavailable used in the PDB
 
 	//Very special Flag to hack CassKop reconcile loop - use with really good Care
-	UnlockNextOperation bool `json:"_unlockNextOperation,omitempty"`
+	UnlockNextOperation bool `json:"unlockNextOperation,omitempty"`
 
 	//Define the Capacity for Persistent Volume Claims in the local storage
+	// +kubebuilder:validation:Pattern=^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
```

Comment:
What is that comment ?

Reply:
Based on my understanding of the code `DataQuantity` ends up being treated as a `Quantity` just like the `CPUAndMem.CPU` and `CPUAndMem.CPU` fields which have the same regex. See https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L30 for more details.